volume/store/store.go

```diff
@@ -369,7 +369,7 @@ func (s *VolumeStore) create(name, driverName string, opts, labels map[string]st
 	vd, err := volumedrivers.CreateDriver(driverName)
 
 	if err != nil {
-		return nil, &OpErr{Op: "create", Name: name, Err: err}
+		return nil, &OpErr{Op: "create driver", Name: name, Err: err}
```

Comment:
I have reverted It seems to have another problem that is unit test is failing
store_test.go
expected := &OpErr{Op: "create", Name: "fakeerror", Err: errors.New("create error")}
"if err != nil && err.Error() != expected.Error() {
t.Fatalf("Expected create fakeError: create error, got %v", err)
}"
I think the initial scheme which add 'create driver' information in createDriver() is better Please give me your advice, Thanks